src/pages/staticPages/webAPI/styles.scss

```diff
@@ -2,4 +2,7 @@
   table {
     margin-bottom:30px !important;
   }
-}
\ No newline at end of file
+}
+a.anchor_no_style {
```

Comment:
i thought this was actually the default in our site.  maybe not

Reply:
this subclass is not used .. it has been dropped